sdk/schemaregistry/schema-registry-avro/package.json

```diff
@@ -0,0 +1,140 @@
+    "build:browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c 2>&1",
+    "build:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c 2>&1",
+    "build:samples": "dev-tool samples prep && cd dist-samples && tsc -p .",
+    "build:test": "tsc -p . && rollup -c rollup.test.config.js 2>&1",
+    "build": "tsc -p . && rollup -c 2>&1 && api-extractor run --local",
+    "check-format": "prettier --list-different --config ../../.prettierrc.json --ignore-path ../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "clean": "rimraf dist dist-* test-dist temp types *.tgz *.log",
+    "execute:js-samples": "npm run build:samples && dev-tool samples run dist-samples/javascript",
+    "execute:ts-samples": "npm run build:samples && dev-tool samples run dist-samples/typescript/dist/dist-samples/typescript/src",
+    "execute:samples": "npm run build:samples && dev-tool samples run dist-samples/javascript dist-samples/typescript/dist/dist-samples/typescript/src",
+    "extract-api": "tsc -p . && api-extractor run --local",
+    "format": "prettier --write --config ../../.prettierrc.json --ignore-path ../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "integration-test:browser": "karma start --single-run",
+    "integration-test:node": "nyc mocha -r esm --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 5000000 --full-trace dist-esm/test/**/*.spec.js",
+    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
+    "lint:fix": "eslint package.json tsconfig.json api-extractor.json src test --ext .ts --fix --fix-type [problem,suggestion]",
+    "lint": "eslint package.json tsconfig.json api-extractor.json src test --ext .ts -f html -o schema-registry-avro-lintReport.html || exit 0",
+    "pack": "npm pack 2>&1",
+    "prebuild": "npm run clean",
+    "test:browser": "npm run build:test && npm run unit-test:browser && npm run integration-test:browser",
+    "test:node": "npm run build:test && npm run unit-test:node && npm run integration-test:node",
+    "test": "npm run build:test && npm run unit-test && npm run integration-test",
+    "unit-test:browser": "karma start --single-run",
+    "unit-test:node": "mocha --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 1200000 --full-trace dist-test/index.node.js",
+    "unit-test": "npm run unit-test:node && npm run unit-test:browser"
+  },
+  "files": [
+    "dist/",
+    "dist-esm/src/",
+    "types/schema-registry-avro.d.ts",
+    "README.md",
+    "LICENSE"
+  ],
+  "//metadata": {
+    "constantPaths": [
+      {
+        "path": "src/generated/generatedSchemaRegistryClientContext.ts",
+        "prefix": "packageVersion"
+      },
+      {
+        "path": "src/constants.ts",
+        "prefix": "SDK_VERSION"
+      },
+      {
+        "path": "swagger/README.md",
```

Comment:
this surprises me, I thought autorest was smart enough to pull the version from package.json so we didn't need to specify it in the swagger MD

Reply:
This one is actually copy and pasted unnecessarily into schema-registry-avro from schema-registry. We don't have the version in this lib, or any swagger. So I will remove it here.

But I'll follow up on schema-registry. I seem to recall playing trying to run autorest with no version in README, and it didn't work as I'd hoped so I opted to add another thing to update here. I'll try again to confirm.

Reply:
When I remove package-version from autorest README and regenerate it overwrites the version with 1.0.0. I'm going to move on and leave it this way, but if there's a step I'm missing to make it work without package-version in README, let me know.

Reply:
Weird. Can you file an issue so we can investigate? 

/cc @joheredi

Reply:
Sure. Should it go in this repo or elsewhere?

Reply:
This repo is fine! ❤️

Reply:
Sorry for the delay, filed #11590